.changeset/empty-mails-sparkle.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/plugin-techdocs': minor
```

Comment:
Should be `patch` since no breaking changes on upgrade. :)
```suggestion
'@backstage/plugin-techdocs': patch
```

Reply:
Ah, thanks